collaboration/README.md

```diff
@@ -1,102 +1,61 @@
-and heard
+  and heard
+- Address the project, not the person - creates a safe environment for achieving
+  common project goal
 
-- Address the project not the person - creates a safe environment for achieving the
-common project goal
-
-</details>
-
-<details>
-<summary> What we do if that trust is broken  </summary>
+## What we do if that trust is broken
 
 - Address it by talking it over with the person who broke it.
-
-- Involve a third party team member if still not addressed
-
-- If a sincere apology is given, accept it, and reset mindset to assuming team
-has positive intentions for everyone
-
+- Involve a third-party team member if still not addressed.
+- If a sincere apology is given, accept it, and reset mindset to assuming the team
+  has positive intentions for everyone.
 - If the trust cannot be repaired, reach out to program staff to have you removed
-from the situation
-
-</details>
+  from the situation.
 
-<details>
-<summary> How we balance discussions vs debate. When is one more appropriate than
-the other?
-  </summary>
+## How we balance discussions vs debate. When is one more appropriate than the other?
 
 - We may debate over which programming language to use or how a function is
-written. Secondly, we may debate over meeting time or places. Debates are healthy
-and inevitable, but discussions MUST follow afterwards.
-
+  written. Secondly, we may debate over meeting time or places. Debates are healthy
+  and inevitable, but discussions MUST follow afterward.
 - Discussion: it is important for discussion to always follow a debate. This is
-when everyone articulates their thoughts on why they disagree or agree, and a
-consensus is reached. If a consensus cannot be reached, then the object of
-disagreement should be put up for vote.
-
-- Aside contentious issues as laid out above, team should be able to discuss things
-normally and agree on what works for everyone
-
-</details>
-
-<details>
-<summary> How the team approaches meeting times and agendas
-
-  </summary>
+  when everyone articulates their thoughts on why they disagree or agree, and a
+  consensus is reached. If a consensus cannot be reached, then the object of
+  disagreement should be put up for a vote.
+- Aside from contentious issues as laid out above,
+  the team should be able to discuss things
+  normally and agree on what works for everyone.
 
-- This is currently up for vote. Check back for final update
+## How the team approaches meeting times and agendas
 
-</details>
+- This is currently up for vote. Check back for final update.
 
-<details>
-<summary> How and where team communicates about different aspects of the project?
-  </summary>
+## How and where the team communicates about different aspects of the project?
 
-- Slack and Whatsapp are the two major means of communication for the team
+- Slack and Whatsapp are the two major means of communication for the team.
 
-</details>
-
-<details>
-<summary> Team's Respect Codes
-  </summary>
```

Comment:
# Explanation for Formatting Changes

The formatting was updated to replace `<details>` and `<summary>` HTML tags with Markdown headings and lists. This ensures compliance with Markdown linting rules and improves compatibility across different Markdown parsers.